packages/react-core/src/components/ToggleGroup/examples/ToggleGroup.md

```diff
@@ -13,22 +13,51 @@ import ShareSquareIcon from '@patternfly/react-icons/dist/esm/icons/share-square
 
 ## Examples
 
-### Default with multiple selectable
-```ts file="./ToggleGroupDefaultMultiple.tsx"
-```
+### Single select toggle group
+
+A single select toggle group allows users to toggle between multiple items.
+
+To support single selection, use the `onChange` callback to pass the selected `<ToggleGroupItem>` to `isSelected`.
```

Comment:
For this and the multi select example, maybe the key point should be more that `isSelected` needs to be true. The onChange is sort of facilitating that in these examples, but an item could be selected just by passing `isSelected={true}` or use something other than the onChange to update an item's selected state.

Maybe something along the lines of:

```suggestion
To indicate whether a `<ToggleGroupItem>` is selected or not, use the `isSelected` property.
```

Reply:
ah, ty this is very helpful! Your suggestion sounds great